docs/tutorials/01_introductory/viz_solar_system.py

```diff
@@ -132,7 +108,7 @@ def init_planet(filename):
 
 
 def get_orbit_period(radius):
-    return np.sqrt(2*np.pi * np.power(radius, 3)/miu)
+    return 2 * np.pi * np.sqrt(np.power(radius, 3)/miu)
```

Comment:
The parentheses in line 118 are redundant and leaving them may raise a PEP warning. Can you remove them?